docs/spec/openapi.yml

```diff
@@ -700,6 +700,107 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+
+  '/namespaces/{namespace}/database-clusters/{dbName}/data-import-jobs':
+    x-everest-resource-name: data-import-jobs
+    get:
+      tags:
+      - Database Cluster
+      summary: List data import jobs for a database cluster
+      description: |
+        This API lists all data import jobs for the database cluster specified by the `name` and `namespace`.
+      operationId: listDataImportJobs
+      parameters:
+        - name: namespace
+          in: path
+          description: Name of the namespace
+          required: true
+          schema:
+            type: string
+        - name: dbName
+          in: path
+          description: Name of the database cluster. Can be found under Metadata["name"] of the DatabaseCluster object.
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DataImportJobList'
+        '500':
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+  '/namespaces/{namespace}/database-clusters/{dbName}/secret':
+    post:
+      tags:
+        - Database Cluster
+      summary: Create a secret for the given database cluster
+      description: |
+        This API creates a secret for the database cluster specified by the `name` and `namespace`
+      operationId: createDatabaseClusterSecret
+      parameters:
+        - name: dbName
+          in: path
+          description: Name of the database cluster
+          required: true
+          schema:
+            type: string
+        - name: namespace
+          in: path
+          description: Name of the namespace
+          required: true
+          schema:
+            type: string
+        - name: secretName
+          in: query
+          description: Optional name of the secret to be created. If not provided, a random name will be generated.
+          required: false
+          schema:
+            type: string
+        - name: dbEngineType
```

Comment:
what is the relation between the secret and DBEngine? it has reference to dbName

Reply:
TBH - the reason we are setting `engineType` in Secret is so that it can work with PG. But as it turns out, we don't require the user to create Secret for the PG data-import. We can get rid of this param for now to avoid confusion ...